portfolio/src/main/webapp/script.js

```diff
@@ -22,7 +22,83 @@ function addRandomGreeting() {
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
-  // Add it to the page.
+
+        {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+}
+
+function testKey() {
+    var inputVal = document.getElementById("myInput").value;
+    if(inputVal == "I am Justice I am God" || inputVal == "i am justice i am God" || inputVal == "I am justice I am kira" || inputVal == "I am justice, I am God")
+        {
+            window.alert("You got it right! My favorite TV show is Deathnote!");
+        }
+    else
+        {
+            window.alert("Try again, you almost got it!");
+        }
+}
+/*
```

Comment:
It's best not to leave commented-out code in a Pull Request. If it's not in use, then its best just to delete it. If you're still working on it, then you can commit it on a development branch.

Reply:
Okay, I deleted the commented out code as I don't plan on developing it anymore.